src/Controller/ActivityController.php

```diff
@@ -275,7 +275,7 @@ public function getActivityDetails(Activity $activity): Response
         $user = $this->getUser();
         $rights = $this->accessRightsPolicy->canAccessActivity($activity, $user);
 
-        if ($rights === false) {
+        if ($rights === false && $user->getRoles() !== array('ROLE_ADMIN')) {
```

Comment:
Use `$this->isGranted('ROLE_ADMIN');`

Reply:
Outside of controllers use `Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface::isGranted()`